### ISSUE TITLE
Javascript Error: Can't find variable: waitForElm

### ISSUE THREAD

**Issue by richhayler** [engram-design/CPNav#104]
**Description**
Looking at the web inspector within the control panel shows a javascript error associated with cp-nav.js "Can't find variable: waitForElm"

<img width="472" alt="CleanShot 2022-06-21 at 12 38 20@2x" src="https://user-images.githubusercontent.com/2505643/174704730-ffefb0f1-b9e6-4b25-b0a4-90bbc8a67042.png">

**Additional info**
- Plugin version: 4.0.2
- Craft version: 4.0.4
- Multi-site: Yes

**Comment by engram-design** [comment]
Hmm, this JS should be rendered pretty much immediately when loading the CP base layout.

Is this happening for all pages, or just some? When reloading the page does it happen consistently?

**Comment by engram-design** [comment]
Fixed for the next release. To get this fix early, change your `verbb/cp-nav` requirement in `composer.json` to:

```
"require": {
  "verbb/cp-nav": "dev-craft-4 as 4.0.2",
  "...": "..."
}
```

Then run `composer update`.

**Comment by engram-design** [comment]
Fixed in [4.0.3](https://github.com/verbb/cp-nav/releases/tag/4.0.3)
